presto-main/src/main/java/com/facebook/presto/server/security/SpnegoFilter.java

```diff
@@ -165,7 +165,8 @@ public Principal getUserPrincipal()
                         return;
                     }
                 }
-                catch (GSSException e) {
+                catch (Exception e) {
+                    LOG.debug(e, "Authentication failed for token %s", parts[1]);
```

Comment:
Is the rethrown exception logged anywhere? If so, would it make sense to rewrite the propagated exception message to include the token? i.e.
```java
throw new RuntimeException("Authentication error for token: " + parts[1], e);